scripts/ci-build.js

```diff
@@ -0,0 +1,83 @@
+const { spawn } = require('./spawn')
+const path = require('path')
+
+const base = path.join(__dirname, '../')
+process.chdir(base)
+async function main() {
+    const prepareCommands = getCommands(`
+    yarn install --link-duplicates --frozen-lockfile
+    yarn lint:report
+    ${
```

Comment:
………………

Why the comment? And if you think that check is worth doing, you might want to ask why you are building at all…